config/webpack.config.react.js

```diff
@@ -131,6 +133,44 @@ module.exports = {
       ],
     }),
     new ContextReplacementPlugin(/moment[/\\]locale$/, new RegExp(langRegex)),
+    new WebpackPwaManifest({
+      name: 'Lisk Hub',
+      short_name: 'Lisk Hub',
+      description: '🖥 Graphical User Interface (GUI) to manage Lisk IDs, tokens and your sidechains ',
+      theme_color: '#005a90',
+      background_color: '#013165',
```

Comment:
The background_color on splash screen IMO doesn't look too good:
![screenshot_2018-04-09-15-08-27-698](https://user-images.githubusercontent.com/1254342/38502257-38d07934-3c0f-11e8-9b48-84b52a581a53.jpeg)
What about using something like what we use when non-pwa is loading?
![screenshot_2018-04-09-15-11-46-605](https://user-images.githubusercontent.com/1254342/38502304-51796414-3c0f-11e8-94e2-e0c798872a27.jpeg)

Reply:
@slaweet already provide some requests to LiskHQ designers to get splash screen image and correct colors palet.